setup.py

```diff
@@ -10,16 +10,16 @@
 
 install_requires = [
     'Django>=1.8,<1.12',
-    'djangorestframework==3.6.4', # Latest version that supports both Django 1.8 and 1.11
+    'djangorestframework==3.6.4',  # Latest version that supports both Django 1.8 and 1.11
     'elasticsearch>=2.4.1,<3',
-    'requests>=2.14,<2.15',
-    'urllib3>=1.21,<1.22',
+    'requests>2.19,<2.20',
+    'urllib3>1.21,<1.24',
     'django-localflavor>=1.5,<1.6',
-    'wagtail-flags>=2.0.5,<2.2',
+    'wagtail-flags>=3.0.0,<3.1',
 ]
 
 testing_extras = [
-    'coverage==4.5.1',
+    'coverage>=4.5.1,<4.6',
```

Comment:
It doesn't seem like we're doing anything particularly advanced with `coverage` here besides just running it and generating a report. I guess we'd want the earliest version that supports the formatting in the [`.coveragerc`](https://github.com/cfpb/ccdb5-api/blob/master/.coveragerc). I might put `>=4.5.1,<4.6` but would have the same question about semantic versioning as above.

Reply:
I don't quite understand your comment. It sounds like you would either (a) use what I have here, or (b) extend the lower bound to include the earliest version that supports our `.coveragerc` (I didn't have an easy time finding that in the [changelog](https://coverage.readthedocs.io/en/coverage-4.5.1a/changes.html), but it might be there), and/or (c) extend the upper bound to include `5.0`. 

In general, I don't know how much to value safety (keeping the constraints narrow so things don't break) vs maintainability (keeping the constraints wider so we don't have to update them all the time), so I'm asking for concrete feedback on what to do.

Also, I didn't consider this question before: Do these testing requirements even get pulled in by cfgov-refresh? I'm wondering how/when it could cause problems if there is a version mismatch in testing library versions.

Reply:
Agh, sorry for my confusing comment! Re-reading this I think I meant to type `>=4.5.1,<5.0` but for some reason didn't type that.

I don't know that I have super strong feelings either way. If you're asking for a concrete direction I think I'd lean on the side of keeping the requirements loose (within the major range) but could easily be convinced that we should pin them for preciseness instead. I can think of good arguments either way.

Testing requirements don't get pulled in by cfgov-refresh. The [`extras_require`](https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies) setting [here](https://github.com/cfpb/ccdb5-api/blob/master/setup.py#L50) means that a plain `pip install` won't install them unless `[testing]` is included in the install command.